node/src/components/consensus/protocols/highway.rs

```diff
@@ -648,19 +651,29 @@ impl<I: NodeIdT, C: Context + 'static> HighwayProtocol<I, C> {
             // We're ahead.
             match state.panorama().get(vid) {
                 None => {
-                    error!(?vid, "received a request for non-existing validator");
```

Comment:
Shouldn't that be really impossible now, with the `==` check? (Same for the next error below.)

Reply:
It should be impossible but I have to pattern match on it to avoid compiler errors. Also, I didn't want to leave it on `ERROR` as we have prometheus alerts on this that wake up our SRE.